pkg/deployments/package.json

```
@@ -33,7 +33,7 @@
     "lint:solidity": "solhint 'tasks/**/*.sol'",
     "lint:typescript": "NODE_NO_WARNINGS=1 eslint . --ext .ts --ignore-path ../../.eslintignore  --max-warnings 0",
     "prepack": "yarn build",
-    "test": "yarn build && hardhat test ./**/test/*.ts",
+    "test": "yarn build && hardhat test ./**/20230410-silo*/**/test/*.ts",
```

```suggestion
    "test": "yarn build && hardhat test ./**/test/*.ts"
```
Can't have task-specific paths in the general package